spec/server_spec.rb

```diff
@@ -0,0 +1,61 @@
+require File.expand_path('../spec_helper', __FILE__)
+require 'server'
+require 'octokit'
+require 'api'
+require 'ping_checker'
+require 'signature_verifier'
+require 'rspec'
+require 'rack/test'
+require 'sinatra'
```

Comment:
Ah neat, I thought they were like Swift/ObjC's import for scope, but they're more "require to be loaded into the runtime" then?